.github/workflows/image.yaml

```diff
@@ -102,6 +102,7 @@ jobs:
         driver: 
           - 535
           - 550
+          - 570
```

Comment:
Question: do we also need to update the version matrix in `precompiled.yaml` (our scheduled pipelines)?

Reply:
Ah yes, thanks for catching that. I will update that as well